### PR TITLE
UI Create new user #1

### PR DESCRIPTION
**Pull request by PaulGrishunin**

### CODE REVIEW

File: RegistrationView.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            'Password:', this.state.password);
+
+        let addUser = {                      //let login, email, pass from forms
+            login: this.state.login,
+            email: this.state.email,
+            password: this.state.password
+        };
+        axios.post('../../../server/RegistrationForm', addUser )    //Send Login, email, pass to server
+
+        axios.get('http://localhost:3000/register')                //Check HTTP status code
```
Comment (wiktorks) [inline RegistrationView.js:33]: Unnecessary GET request, HTTP status code should be checked in POST request

File: RegistrationView.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        console.log('form is submitted. Login:', this.state.login,
+            'Email:', this.state.email,
+            'Password:', this.state.password);
+
+        let addUser = {                      //let login, email, pass from forms
+            login: this.state.login,
+            email: this.state.email,
+            password: this.state.password
+        };
+        axios.post('../../../server/RegistrationForm', addUser )    //Send Login, email, pass to server
```
Comment (wiktorks) [inline RegistrationView.js:31]: URL for post request has to be changed.

File: RegistrationView.js (hunk 1 above)
Comment (SzymonSmykala) [reply RegistrationView.js:33]: generally do not hardcode URI in application http://localhost:3000/ 
It should be passed in npm run parameter.

File: RegistrationView.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleSubmit(event){
+        event.preventDefault();
+        console.log('form is submitted. Login:', this.state.login,
+            'Email:', this.state.email,
+            'Password:', this.state.password);
+
+        let addUser = {                      //let login, email, pass from forms
```
Comment (SzymonSmykala) [inline RegistrationView.js:26]: remove all comments

File: RegistrationView.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        };
+        axios.post('../../../server/RegistrationForm', addUser )    //Send Login, email, pass to server
+
+        axios.get('http://localhost:3000/register')                //Check HTTP status code
+            .then(result => console.log('success:', result))
+            .catch(error => console.log('error:', error));
+    };
+
+    handleEmailChange(event) {
+        console.log('email was changed', event.target.value);
```
Comment (SzymonSmykala) [inline RegistrationView.js:39]: remove console.log

**Comment by SzymonSmykala** [comment]
Add unit tests

**Comment by ZPoreba** [comment]
file
Why files are placed outside of directory client ?

**Comment by ZPoreba** [comment]
And why you created another package.json ?

**Comment by ZPoreba** [comment]
Please adjust code to changes from backend (branch registration). Make sure that endpoints work, add handling error messages returned from backend, which should also be displayed to user.

**Pull request closed without merging**